cmd/root.go

```diff
@@ -58,7 +58,7 @@ func Init(version, artifactArch string) {
 
 	authInit()
 	mergeInit()
-	overlayInit()
+	addCommand(rootCmd, overlayCmd)
```

Comment:
Why not just make an Init for overlay?

Reply:
Purely stylistic

Reply:
This is how we initialize commands using the new interface. Once every command is migrated over, we can change `root` to be a `CommandGroup`, in which case we won't need to do any of this